src/form/acp.jl

```diff
@@ -149,7 +149,53 @@ function constraint_ohms_yt_to{T <: AbstractACPForm}(pm::GenericPowerModel{T}, f
     t_to = pm.var[:t][t_bus]
 
     c1 = @NLconstraint(pm.model, p_to == g*v_to^2 + (-g*tr-b*ti)/tm*(v_to*v_fr*cos(t_to-t_fr)) + (-b*tr+g*ti)/tm*(v_to*v_fr*sin(t_to-t_fr)) )
-    c2 = @NLconstraint(pm.model, q_to == -(b+c/2)*v_to^2 - (-b*tr+g*ti)/tm*(v_to*v_fr*cos(t_fr-t_to)) + (-g*tr-b*ti)/tm*(v_to*v_fr*sin(t_to-t_fr)) )
+    c2 = @NLconstraint(pm.model, q_to == -(b+c/2)*v_to^2 - (-b*tr+g*ti)/tm*(v_to*v_fr*cos(t_to-t_fr)) + (-g*tr-b*ti)/tm*(v_to*v_fr*sin(t_to-t_fr)) )
+    return Set([c1, c2])
+end
+
+"""
+Creates Ohms constraints for shiftable PSTs (yt post fix indicates that Y and T values are in rectangular form)
+
+```
+p[f_idx] == g*v_fr^2 + (-g)*(v_fr*v_to*cos((t_fr - t_shift_fr) - (t_to - t_shift_to))) + (-b)*(v_fr*v_to*sin((t_fr - t_shift_fr) - (t_to - t_shift_to)))
+q[f_idx] == -(b+c/2)*v_fr^2 - (-b)*(v_fr*v_to*cos((t_fr - t_shift_fr) - (t_to - t_shift_to))) + (-g)*(v_fr*v_to*sin((t_fr - t_shift_fr) - (t_to - t_shift_to)))
+```
+"""
+function constraint_ohms_yt_from_shiftable{T <: AbstractACPForm}(pm::GenericPowerModel{T}, f_bus, t_bus, f_idx, t_idx, g, b, c)
+    p_fr = pm.var[:p][f_idx]
+    q_fr = pm.var[:q][f_idx]
+    v_fr = pm.var[:v][f_bus]
+    v_to = pm.var[:v][t_bus]
+    t_fr = pm.var[:t][f_bus]
+    t_to = pm.var[:t][t_bus]
+    t_shift_fr = pm.var[:t_shift][f_idx]
+    t_shift_to = pm.var[:t_shift][t_idx]
+
+    c1 = @NLconstraint(pm.model, p_fr == g*v_fr^2 + (-g)*(v_fr*v_to*cos((t_fr - t_shift_fr) - (t_to - t_shift_to))) + (-b)*(v_fr*v_to*sin((t_fr - t_shift_fr) - (t_to - t_shift_to))))
```

Comment:
I see.  My concern is then that the optimization model formulation should match the data format.  The Matpower assumes a simpler pi-model.  So, if you want to develop a network model based on Fig. 2.9 you should also extend the data format to explicitly model this other type of pi-model.

For what its worth, in the past I have had success building models with OLTP by making Matpower's `ratio` and `shift` parameters decision variables (e.g. see, http://sites.ieee.org/psace-mho/panels-and-competitions-2014-opf-problems/).  I cannot immediately see the added value of the "transformer symmetrical model", but this is byond my knowledge of Power Engineering.  Maybe you can provide some motivation for this model over the Matpower one?